### PR TITLE
feat: zetasql syntax support, logic op & create table statement

### LINKED ISSUE #99: Add syntax op `&&`, `||`, `%`, `MOD`, `DIV`, `!`
**Is your feature request related to a problem? Please describe.**

**Describe the solution you'd like**

**Describe alternatives you've considered**

**Additional context**

### LINKED ISSUE #103: Support create table statement
**Is your feature request related to a problem? Please describe.**

**Describe the solution you'd like**

**Describe alternatives you've considered**

**Additional context**

### PR DESCRIPTION
**Pull request by aceforeverd**
+ close #99, support logic operators:
   - `!`
   - `div`: integer division
   - `%`, `mod`
+ update cmakelists: update bison & protoc path
+ close #103, support create table statement

### CODE REVIEW

**Comment by jingchen2222** [comment]
Please update libzetasql

File: src/planv2/ast_node_converter.cc [hunk 1]
```diff
@@ -816,5 +833,408 @@ base::Status ConvertQueryNode(const zetasql::ASTQuery* root, node::NodeManager*
...
+
+    switch (element->node_kind()) {
+        case zetasql::AST_COLUMN_DEFINITION: {
+            auto column_def = element->GetAsOrNull<zetasql::ASTColumnDefinition>();
+            if (nullptr == column_def) {
+                status.msg = "column definition is null";
+                status.code = common::kPlanError;
+                return status;
+            }
+            auto not_null_columns = column_def->schema()->FindAttributes<zetasql::ASTNotNullColumnAttribute>(
```
[the remark this answers is not in the document]
Comment (aceforeverd) [reply src/planv2/ast_node_converter.cc:901]: it seems reasonable. https://github.com/jingchen2222/zetasql/blob/feat/hybridse-zetasql/zetasql/parser/bison_parser.y#L2888

**Comment by aceforeverd** [comment]
TODO: add CHANGELOG

**Review by jingchen2222 — APPROVED** [review]
LGTM

**Pull request merged**